src/main/controllers/tradeManager.controller.ts

```
@@ -180,6 +187,22 @@ export class TradeManagerController {
     }
   }
 
+  public async changeCSFloatState(
+    newState: boolean,
+    username: string
+  ): Promise<void> {
+    const tradeManager = this.tradeManagers.get(username);
+    if (!tradeManager) throw new Error("User not found");
+    try {
+      if (newState) return tradeManager.startMarketcsgoClient();
+      return tradeManager.stopMarketcsgoClient();
+    } catch (err) {
+      if (!newState) return tradeManager.startMarketcsgoClient();
```

this function should throw the error and not return anything inside catch statement